helpers/tailwind-css-cheat-sheet(v4-and-v3).json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "Tailwind CSS Cheat Sheet(v4 and v3)",
+  "desc": "A comprehensive Tailwind CSS cheat sheet to instantly find all utility classes and their CSS properties in one place.",
```

Comment:
Please follow the contribution guidelines (that you deleted). 🫣

https://github.com/stefanjudis/tiny-helpers/blob/primary/.github/pull_request_template.md

> the desc field includes an "actionable sentence" (e.g. "Create something great" or "Transform something into something else").